src/media/js/views/tutorial.js

```diff
@@ -3,9 +3,28 @@ define('views/tutorial',
     function(l10n, z, smartButton, SpatialNavigation) {
     var gettext = l10n.gettext;
 
+    var imageLoader = document.createElement('img');
+
+    // Ensure background image is loaded.
+    function loadBackgroundImage($slide, callback) {
+        imageLoader.src = $slide.find('.slide-image')
+                                .css('background-image')
+                                .replace(/url\(['"]?(.*?)['"]?\)/i, '$1');
+
+        imageLoader.onload = function () {
+            $slide.removeClass('invisible');
+
+            callback();
+        };
```

Comment:
We should add an `onerror` and pretty much run the same function as the `onload` - or create a background CSS gradient as a filler. If the image 404s or something right now the content will never show up.